KoikatuGameplayMod/Hooks/HSceneHooks.cs

```diff
@@ -14,6 +14,7 @@ public static void ApplyHooks(Harmony instance)
         }
 
         #region Keep first time status if chara is still a virgin
+        //Not needed and has no effect in free H or official VR
```

Comment:
What do you mean by not needed? Only it freeh/vr or overall?

Reply:
I put that comment there to clarify that since those hooks are not needed for free H, we don't need to patch the equivalent methods for official VR, in which free H is the only mode available.